src/components/Sidebar/Sidebar.js

```diff
@@ -58,20 +59,35 @@ const openStyles = ({ theme, open }) =>
 
 const Drawer = styled('nav')(baseStyles, openStyles);
 
-const Sidebar = ({ children, open, closeButtonLabel, onClose, ...props }) => (
-  <Fragment>
-    <Drawer open={open} {...props}>
-      {children}
-    </Drawer>
-    <Backdrop visible={open} onClick={onClose} data-testid="sidebar-backdrop" />
-    <CloseButton
-      visible={open}
-      label={closeButtonLabel}
-      onClick={onClose}
-      data-testid="sidebar-close-button"
-    />
-  </Fragment>
-);
+const Sidebar = ({
+  children,
+  open,
+  closeButtonLabel,
+  onClose,
+  tracking,
+  ...props
+}) => {
+  const handleClose = useClickTracker(onClose, tracking, 'sidebar-close');
```

Comment:
I think it's best to move the label values to static values for better understanding.

Reply:
Do you mean something like `const COMPONENT_NAME = 'sidebar-close'` and use it here?